src/main/java/frc/robot/commands/shooter/ManualShooter.java

```diff
@@ -38,9 +39,10 @@ public void initialize() {
   @Override
   public void execute() {
     m_shooter.set(Constants.Shooter.IDLE_SPEED * Constants.Shooter.VELOCITY_FF); // Run at resting speed
-    if (m_joystick.getPOV() == 0 || m_joystick.getPOV() == 45 || m_joystick.getPOV() == 315) {
+    if (m_POVSupplier.getAsDouble() == 0 || m_POVSupplier.getAsDouble() == 45 || m_POVSupplier.getAsDouble() == 315) {
       m_shooter.setSetpoint(m_shooter.getSetpoint() + 50);
-    } else if (m_joystick.getPOV() == 180 || m_joystick.getPOV() == 135 || m_joystick.getPOV() == 225) {
+    } else if (m_POVSupplier.getAsDouble() == 180 || m_POVSupplier.getAsDouble() == 135
+        || m_POVSupplier.getAsDouble() == 225) {
```

Comment:
Not quite what we're looking for, the intent was more to have `m_shooter.setSetpoint(m_setpointSupplier.getAsDouble())`, and then pass the POV logic as an argument

Reply:
I'm not sure I understand. Which argument do I pass the POV logic to.

Reply:
Resolved in commit 7dc8382